web/frontend/src/components/slam.vue

```diff
@@ -169,17 +190,30 @@ const executeMoveOnMap = async () => {
     new grpc.Metadata(),
     (error: ServiceError | null, response: motionApi.MoveOnMapResponse | null) => {
       if (error) {
-        moveClick = !moveClick;
+        allowMove = !allowMove;
+        executeDeleteDestinationMarker();
         toast.error(`Error moving: ${error}`);
         return;
       }
-      moveClick = !moveClick;
+      allowMove = !allowMove;
+      executeDeleteDestinationMarker();
```

Comment:
remove marker if once we complete the MoveOnMap request, regardless of success or failure

Reply:
I looked through every document associated with this project so far, and I can't find any requirement to remove the destination marker if the move fails. Just so I know, does this have a product requirement associated with it that I am not aware of?

Reply:
> does this have a product requirement associated with it that I am not aware of?

no, I will have to check in with product to ask if this is a flow they think is good.

Reply:
just checked in with product about this.
They agree that removing the destination marker after success or failure is the right flow of events.